<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonWidget</class>
 <widget class="QWidget" name="PersonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>109</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
#PersonWidget
{
	border-right:1px solid rgb(170,170,170);
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 90, 140, 255), stop:1 rgba(0, 180, 250, 255));
}
#PersonWidget:hover
{
     background: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, x3:0, y3:1,stop:0 rgba(0, 90, 140, 255), stop:0.5 rgba(130, 200, 250, 255),stop:0.8 rgba(0, 180, 250, 255));

}
#PersonWidget QLabel
{
	font-size:11px;
	font-weight:normal;
	border:none;
	color:rgb(240,240,240);
}
#PersonWidget QPushButton
{
	font-size:12px;
	font-weight:normal;
	color:rgb(240,240,240);
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 90, 140, 255), stop:1 rgba(0, 180, 250, 255));
	border:1px groove rgba(0, 90, 140, 255);
	border-radius:8px;
	padding:4px 12px;
}
#PersonWidget QPushButton:hover
{
     background: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, x3:0, y3:1,stop:0 rgba(0, 90, 140, 255), stop:0.5 rgba(130, 200, 250, 255),stop:0.8 rgba(0, 180, 250, 255));

}
#PersonWidget QPushButton:pressed
{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(130, 200, 250, 255), stop:1 rgba(130, 200, 250, 255));
	color:rgb(0,0,0);
}
#PersonWidget QToolButton
{
	font-weight:normal;
	color:rgb(240,240,240);
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 90, 140, 255), stop:1 rgba(0, 180, 250, 255));
	border:1px groove rgba(0, 90, 140, 255);
	border-radius:8px;
	padding:1px 1px;
}
#PersonWidget QToolButton:hover
{
     background: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, x3:0, y3:1,stop:0 rgba(0, 90, 140, 255), stop:0.5 rgba(130, 200, 250, 255),stop:0.8 rgba(0, 180, 250, 255));

}
#PersonWidget QLabel:hover
{
	background-color: rgb(105, 223, 255);
	color:rgb(0,0,0);

}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chb_peronSelect">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>13</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btn_personMan">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4">
      <widget class="QLabel" name="lb_personAvatar">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>65</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/images/images/buddy-green.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="lb_personIp">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="lb_personName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn_personChat">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Chat</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
